abis/linux/input.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _ABIBITS_INPUT_sH
+#define _ABIBITS_INPUT_sH
+
+#define BTN_MOUSE 0x110
```

Comment:
We should not ship this, this is part of `input-event-codes.h` of the linux kernel headers. Programs should grab the values from there. Mlibc code in the linux option can make use of those headers too.